corehq/apps/reports/views.py

```diff
@@ -425,7 +425,8 @@ def hq_update_saved_export(req, domain):
     index = int(req.POST['index'])
     group_config = get_document_or_404(HQGroupExportConfiguration, domain, group_id)
     config, schema = group_config.all_exports[index]
-    rebuild_export_task.delay(group_id, index)
+    filter = SerializableFunction(instances)
```

Comment:
Custom exports run through this function: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/views.py#L293 Which creates the filter on line 304. (the view they use is in that file on line 274.